### ISSUE TITLE
Build fails due to inaccessible palantir dependencies

### ISSUE THREAD

**Issue by snazy** [apache/iceberg#2462]
Build is failing with this message:
```
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not resolve com.palantir.baseline:gradle-junit-reports:3.36.2.
     Required by:
         project : > com.palantir.baseline:gradle-baseline-java:3.36.2
      > Could not resolve com.palantir.baseline:gradle-junit-reports:3.36.2.
         > Could not get resource 'http://palantir.bintray.com/releases/com/palantir/baseline/gradle-junit-reports/3.36.2/gradle-junit-reports-3.36.2.pom'.
            > Could not GET 'http://palantir.bintray.com/releases/com/palantir/baseline/gradle-junit-reports/3.36.2/gradle-junit-reports-3.36.2.pom'. Received status code 403 from server: Forbidden
   > Could not resolve com.palantir.javaformat:palantir-java-format-spi:0.3.9.
     Required by:
         project : > com.palantir.baseline:gradle-baseline-java:3.36.2
      > Could not resolve com.palantir.javaformat:palantir-java-format-spi:0.3.9.
         > Could not get resource 'http://palantir.bintray.com/releases/com/palantir/javaformat/palantir-java-format-spi/0.3.9/palantir-java-format-spi-0.3.9.pom'.
            > Could not GET 'http://palantir.bintray.com/releases/com/palantir/javaformat/palantir-java-format-spi/0.3.9/palantir-java-format-spi-0.3.9.pom'. Received status code 403 from server: Forbidden
   > Could not resolve com.palantir.javaformat:gradle-palantir-java-format:0.3.9.
     Required by:
         project : > com.palantir.baseline:gradle-baseline-java:3.36.2
      > Could not resolve com.palantir.javaformat:gradle-palantir-java-format:0.3.9.
         > Could not get resource 'http://palantir.bintray.com/releases/com/palantir/javaformat/gradle-palantir-java-format/0.3.9/gradle-palantir-java-format-0.3.9.pom'.
            > Could not GET 'http://palantir.bintray.com/releases/com/palantir/javaformat/gradle-palantir-java-format/0.3.9/gradle-palantir-java-format-0.3.9.pom'. Received status code 403 from server: Forbidden
```

Just changing `http` to `https` doesn't change anything.

Bumping to `com.palantir.baseline:gradle-baseline-java:3.76.0` helps a little, but unfortunately, there's still one dependency missing:
```
   > Could not resolve com.palantir.javaformat:palantir-java-format-spi:0.3.9.
     Required by:
         project : > com.palantir.baseline:gradle-baseline-java:3.36.2
      > Could not resolve com.palantir.javaformat:palantir-java-format-spi:0.3.9.
         > Could not get resource 'http://palantir.bintray.com/releases/com/palantir/javaformat/palantir-java-format-spi/0.3.9/palantir-java-format-spi-0.3.9.pom'.
            > Could not GET 'http://palantir.bintray.com/releases/com/palantir/javaformat/palantir-java-format-spi/0.3.9/palantir-java-format-spi-0.3.9.pom'. Received status code 403 from server: Forbidden
   > Could not resolve com.palantir.javaformat:gradle-palantir-java-format:0.3.9.
     Required by:
         project : > com.palantir.baseline:gradle-baseline-java:3.36.2
      > Could not resolve com.palantir.javaformat:gradle-palantir-java-format:0.3.9.
         > Could not get resource 'http://palantir.bintray.com/releases/com/palantir/javaformat/gradle-palantir-java-format/0.3.9/gradle-palantir-java-format-0.3.9.pom'.
            > Could not GET 'http://palantir.bintray.com/releases/com/palantir/javaformat/gradle-palantir-java-format/0.3.9/gradle-palantir-java-format-0.3.9.pom'. Received status code 403 from server: Forbidden
```
(Note: it seems, that the `repositories` block in `build.gradle` can be safely removed.)

Seems that all Palantir artifacts are only published to bintray according to mvnrepository.com , which makes it quite hard to find an alternative Maven repo for `palantir-java-format-spi`.

Looking, whether there's at least a temporary, dirty hack to get it working.

**Comment by snazy** [comment]
Hum - trying to build `palantir-java-format` locally fails with similar errors due to bintray returning HTTP/403 for everything.

**Comment by rdblue** [comment]
Long term, we should probably remove the baseline plugins and not rely on Palantir. Hopefully we can get something working without doing all that work to unblock commits, though.

**Comment by rymurr** [comment]
It looks like this was an intentional outage to test if everyone is moving off bintray: https://status.bintray.com/incidents/4mzph0y8l6hk?u=sqw9s2pztxql

I suppose we can close #2463 for now and we have until the 26th of April to find a permanent solution?

Anyone have any idea of when Palantir will be moving their packages to gradle plugins (some are already there) or should we try and remove baseline in the next 2 weeks?

**Comment by snazy** [comment]
Def. need a new HTTP 5xx status code "Intentional Brownout"

**Comment by wypoon** [comment]
The bintray temporary outage was to raise awareness that bintray is sunsetting for good. While the outage is over now and builds succeed again, I don't think this issue should be closed unless a new one is opened to find a permanent solution (I haven't seen a new issue for that).

**Comment by snazy** [comment]
I don't mind reopening this issue until the problem's fully solved.
